index.html

```diff
@@ -2250,6 +2329,36 @@ <h1>Core Representations</h1>
 for more information.
     </p>
 
+    <section>
+      <h2>Creating Representations</h2>
```

Comment:
Suggest changing this heading to "Representation Requirements" but it's okay to do that in a subsequent PR as well.

Reply:
Done.